src-tauri/src/main.rs

```diff
@@ -148,3 +157,40 @@ impl ApplicationState {
         Self::new()
     }
 }
+
+#[tauri::command]
+async fn get_cloud_token() -> Result<String, String>{
+    Ok(std::env::var(String::from("K6_CLOUD_TOKEN")).unwrap_or(String::from("")))
+}
+
+#[tauri::command]
+fn set_cloud_token(token: String) -> Result<(), String>{
+    std::env::set_var("K6_CLOUD_TOKEN", token);
+    Ok(())
+}
+
+#[tauri::command]
+async fn run_script_in_cloud(script: String, project_id: String) -> Result<String, String>{
+    let mut child = Command::new("k6")
+    .arg("cloud")
+    .arg("-")
+    .stdin(Stdio::piped())
+    .stdout(Stdio::piped())
+    .env("K6_CLOUD_TOKEN", std::env::var(String::from("K6_CLOUD_TOKEN")).unwrap_or(String::from("")))
+    .env("K6_CLOUD_PROJECT_ID", project_id)
+    .env("K6_CLOUD_NAME", "kroco6 script.js")
```

Comment:
There's [this issue](https://github.com/grafana/k6/issues/3077), which was a surprise as I could stream results to cloud without the test name, but couldn't _run_ test in cloud without it